portals/bring-your-own-code-samples/react-sample/README.md

```diff
@@ -0,0 +1,123 @@
+# Power Pages - Bring Your Own Code - React Sample
+
+This is a react based sample application to demonstrate how a react website can be hosted on Power Pages and integrate with features like authentication, authorization, web apis etc.
+
+This is a modern web application for managing car sales, inventory, and customer information.
+
+## Features
+
+- Authentication using Microsoft Entra Id (see [AuthButton.tsx](src/components/AuthButton.tsx) for implementation)
+- Authorization using Power Pages web roles (see [Layout.tsx:45-49](src/components/Layout.tsx#L45-L49) for implementation)
+- Fetching data from external data sources through [virtual tables](https://learn.microsoft.com/en-us/power-pages/configure/virtual-tables) and web apis (see [SalesLeads.tsx](src/pages/SalesLeads.tsx#L30-L63) for implementation)
+
+## Prerequisites
+
+- Node.js (v14 or higher)
+- npm (v6 or higher)
+
+## Installation
+
+1. Clone the repository:
+
+    ```bash
+    git clone https://github.com/microsoft/PowerApps-Samples
+    cd portals/bring-your-own-code-samples\react-sample\
+    ```
+
+1. Install dependencies:
+
+    ```bash
+    npm install
+    ```
+
+## Development
+
+To start the development server:
+
+```bash
+npm run dev
+```
+
+The application will be available at `http://localhost:5173`
+
+## Building for Production
+
+To create a production build:
+
+```bash
+npm run build
+```
+
+The built files will be in the `dist` directory.
+
+## Running on Power Pages
+
+Below steps will help you run this app in Power Pages.
+
+### Setup
+
+1. Install [Microsoft Power Platform CLI](https://learn.microsoft.com/en-us/power-platform/developer/cli/introduction?tabs=windows#install-microsoft-power-platform-cli). (Version should be >= 1.43.6)
```

Comment:
Remove `en-us/` from URLs to documentation on learn.microsoft.com

Reply:
Removed